### ISSUE TITLE
No updates from last 2 months

### ISSUE THREAD

**Issue by vjarora1978** [Microsoft/Mobius#657]
There are no updates from last 2 months, is Microsoft stopping support to this library?

I understand everything is driven by demand, but do we have an future release plan?

or should not we learn this API, what do you suggest?

**Comment by skaarthik** [comment]
Like any open source project, updates to Mobius are driven by the demand in the community and the timeline for an update depends on the availability of contributors and their interest in working that update.

Is there any particular update you are waiting on? It will be useful if you create a new issue for that or add your comments if there is an existing [issue](https://github.com/Microsoft/Mobius/issues) to track it?

**Comment by dsyme** [comment]
> Is there any particular update you are waiting on? It will be useful if you create a new issue for that or add your comments if there is an existing issue to track it?

Or even better send a pull request to implement and test it :)
